scripts/copy_hazelcast_plugin.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# For when you've built the hazelcast plugin locally and want this in your next stack
+cp ../openfire-hazelcast-plugin/target/hazelcast-openfire-plugin-assembly.jar ./plugins_for_clustered/hazelcast.jar
+cp ../openfire-hazelcast-plugin/target/hazelcast-openfire-plugin-assembly.jar ./plugins_for_federated/hazelcast.jar
+cp ../openfire-hazelcast-plugin/target/hazelcast-openfire-plugin-assembly.jar ./plugins_for_other/hazelcast.jar
```

Comment:
Do these folders want the Hazelcast plugin?

Reply:
Fair points. In the latest commit I removed two incorrect copy lines, and I removed the monitoring plugin binary from the default sets.